File: C/BEGINNER/1049 Animal.c
#include <stdio.h>
#include <string.h>

int main()
{
    char structure[13], type[9], diet[11];
    scanf("%s", structure);
    scanf("%s", type);
    scanf("%s", diet);
    if (strcmp(structure, "vertebrado") == 0)
    {
        if (strcmp(type, "ave") == 0)
        {
            if (strcmp(diet, "carnivoro") == 0)
                printf("aguia\n");
            else
                printf("pomba\n");
        }
        else
        {
            if (strcmp(diet, "onivoro") == 0)
                printf("homem\n");
            else
                printf("vaca\n");
        }   
    }
    else
    {
        if (strcmp(type, "inseto") == 0)
        {
            if (strcmp(diet, "hematofago") == 0)
                printf("pulga\n");
            else
                printf("lagarta\n");
        }
        else
        {
            if (strcmp(diet, "hematofago") == 0)
                printf("sanguessuga\n");
            else
                printf("minhoca\n");
        }
    }
    return 0;
}

File: Python/BEGINNER/1049 Animal.py
# op = option

op1 = input()
op2 = input()
op3 = input()

if op1 == 'vertebrado':
    if op2 == 'ave':
        if op3 == 'carnivoro':
            print('aguia')
        else:
            print('pomba')
    else:
        if op3 == 'onivoro':
            print('homem')
        else:
            print('vaca')
else:
    if op2 == 'inseto':
        if op3 == 'hematofago':
            print('pulga')
        else:
            print('lagarta')
    else:
        if op3 == 'hematofago':
            print('sanguessuga')
        else:
            print('minhoca')
